cassandra-commons/src/test/resources/scheduler.yml

```diff
@@ -5,6 +5,8 @@ executor:
   memory_mb : ${EXECUTOR_MEMORY_MB:-768}
   heap_mb : ${EXECUTOR_HEAP_MB:-512}
   api_port : ${EXECUTOR_API_PORT:-9000}
+  network_mode : ${EXECUTOR_NETWORK_MODE:-"host"}
```

Comment:
These configuration options should also be exposed through the `universe/config.json` and `universe/marathon.json.mustache` files.